### PR TITLE
Issue9 password age scan and Issue 10 - Root MFA

### PR DESCRIPTION
**Pull request by romancarve**
Includes the addition of a Root MFA check and password age check, as well as a fix for the findings count.

### CODE REVIEW

File: scanamabob/commands/scan.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 import scanamabob.scans.rds as rds
 import scanamabob.scans.redshift as redshift
 from scanamabob.context import Context, add_context_to_argparse
-
+import IPython
```
Comment (austin-ralls-cs) [inline scanamabob/commands/scan.py:14]: I think you forgot to remove this

File: scanamabob/scans/iam.py [hunk 2]
```diff
@@ -54,6 +56,35 @@ def run(self, context):
...
+                password_last_changed = datetime.strptime(col[5], '%Y-%m-%dT%H:%M:%S+00:00')
+                if (today - password_last_changed).days > self.MAX_PASSWORD_AGE:
+                    users_over_max_password_age.append(col[0])
+            except ValueError:
+                creation_date = datetime.strptime(col[2], '%Y-%m-%dT%H:%M:%S+00:00')
+                if (today - creation_date).days > self.MAX_PASSWORD_AGE:
+                    users_over_max_password_age.append(col[0])
+                    
+        if len(users_over_max_password_age) > 0:
+            print(users_over_max_password_age)
```
Comment (meadori-cs) [inline scanamabob/scans/iam.py:84]: Debugging code?

File: scanamabob/commands/scan.py (hunk 1 above)
Comment (meadori-cs) [reply scanamabob/commands/scan.py:14]: I think it got removed in 4ee3f457ca37696ce1cc1d6e225a07355c1c703d.

Comment (austin-ralls-cs) [reply scanamabob/commands/scan.py:14]: and added again in https://github.com/CarveSystems/scanamabob/pull/35/commits/fc954e33f17ae6c2a0681ebc484a964667eef399

**Comment by CarveKenneth** [comment]
Closes #9

**Pull request merged**